### ISSUE TITLE
Convert to yx or cyx

### ISSUE THREAD

**Issue by LucaMarconato** [glencoesoftware/bioformats2raw#180]
I am converting a yx ome.tif file to NGFF. Is there an option to create a yx or cyx image? I don't need the t and z dimensions.

**Comment by melissalinkert** [comment]
@LucaMarconato : No, there is not an option to remove unused dimensions. bioformats2raw intentionally does not implement all features of the OME-NGFF specification, and assumes that all 5 dimensions should be preserved.

We're unlikely to implement this feature any time soon, but would be happy to look at a pull request if you (or anyone else) are interested in working on it.

**Comment by LucaMarconato** [comment]
Thank you for the information!

**Comment by jluethi** [comment]
Just adding my voice to that of @LucaMarconato & @d-v-b [over here](https://github.com/glencoesoftware/bioformats2raw/issues/209) to mention that it would be very useful to avoid adding such extra dimensions. An example of the issue this raises with other tools can be found in this discussion here: https://forum.image.sc/t/fractal-framework-zarr-compatibility/92536/10?u=jluethi.

It's not our top priority, so can't provide a PR for this anytime soon. But wanted to give a +1 for its usefulness if this issue gets raised in the future.
